order.go

```diff
@@ -171,32 +171,49 @@ type DiscountCode struct {
-	Properties                 []NoteAttribute  `json:"properties,omitempty"`
-	ProductExists              bool             `json:"product_exists,omitempty"`
-	FulfillableQuantity        int              `json:"fulfillable_quantity,omitempty"`
-	Grams                      int              `json:"grams,omitempty"`
-	FulfillmentStatus          string           `json:"fulfillment_status,omitempty"`
-	TaxLines                   []TaxLine        `json:"tax_lines,omitempty"`
-	OriginLocation             *Address         `json:"origin_location,omitempty"`
-	DestinationLocation        *Address         `json:"destination_location,omitempty"`
-	AppliedDiscount            *AppliedDiscount `json:"applied_discount,omitempty"`
+	ID                         int64                 `json:"id,omitempty"`
+	ProductID                  int64                 `json:"product_id,omitempty"`
+	VariantID                  int64                 `json:"variant_id,omitempty"`
+	Quantity                   int                   `json:"quantity,omitempty"`
+	Price                      *decimal.Decimal      `json:"price,omitempty"`
+	TotalDiscount              *decimal.Decimal      `json:"total_discount,omitempty"`
+	Title                      string                `json:"title,omitempty"`
+	VariantTitle               string                `json:"variant_title,omitempty"`
+	Name                       string                `json:"name,omitempty"`
+	SKU                        string                `json:"sku,omitempty"`
+	Vendor                     string                `json:"vendor,omitempty"`
+	GiftCard                   bool                  `json:"gift_card,omitempty"`
+	Taxable                    bool                  `json:"taxable,omitempty"`
+	FulfillmentService         string                `json:"fulfillment_service,omitempty"`
+	RequiresShipping           bool                  `json:"requires_shipping,omitempty"`
+	VariantInventoryManagement string                `json:"variant_inventory_management,omitempty"`
+	PreTaxPrice                *decimal.Decimal      `json:"pre_tax_price,omitempty"`
+	Properties                 []NoteAttribute       `json:"properties,omitempty"`
+	ProductExists              bool                  `json:"product_exists,omitempty"`
+	FulfillableQuantity        int                   `json:"fulfillable_quantity,omitempty"`
+	Grams                      int                   `json:"grams,omitempty"`
+	FulfillmentStatus          string                `json:"fulfillment_status,omitempty"`
+	TaxLines                   []TaxLine             `json:"tax_lines,omitempty"`
+	OriginLocation             *Address              `json:"origin_location,omitempty"`
+	DestinationLocation        *Address              `json:"destination_location,omitempty"`
+	AppliedDiscount            *AppliedDiscount      `json:"applied_discount,omitempty"`
+	DiscoundAllocations        []DiscoundAllocations `json:"discount_allocations,omitempty"`
```

Comment:
There are a few spelling mistakes on line 200 and 203. Can you please replace `Discound` with `Discount`?

Reply:
thanks